### DISCUSSION TITLE
Support to organize queries and dashboards into folders

### DISCUSSION THREAD

**Discussion by danieldestro** [getredash/redash#6055] [Ideas]
I would like to be able to create folders in both Query and Dashboard areas so I can organize my itens better. Tags are very useful but they do not help to organize things as folders can.

**Comment by Jengro777** [comment]
Community/redash deploy

https://github.com/RedashCommunity/redash/discussions/26

**Comment by danieldestro** [comment]
@Avey777 this has nothing to do with my request.
